core/src/main/scala/spinoco/fs2/cassandra/internal/ctype/BooleanCType.scala

```diff
@@ -0,0 +1,50 @@
+package spinoco.fs2.cassandra.internal.ctype
+
+import com.datastax.oss.driver.api.core.ProtocolVersion
+import com.datastax.oss.driver.api.core.`type`.DataType
+import com.datastax.oss.driver.api.core.`type`.codec.TypeCodecs
+import scodec.{Attempt, Codec, DecodeResult, Err, SizeBound}
+import scodec.bits.{BitVector, ByteVector}
+import spinoco.fs2.cassandra.CType
+import spinoco.fs2.cassandra.util
+
+object BooleanCType {
+
+  val instance: CType[Boolean] = {
+    val trueValue = ByteVector.apply[Byte](0x01).bits
+    val falseValue = ByteVector.apply[Byte](0x00).bits
+
+    new CType[Boolean] {
+      def cqlType: DataType = TypeCodecs.BOOLEAN.getCqlType
+
+      def cqlCodec(protocolVersion: ProtocolVersion): Codec[Boolean] = {
+        new Codec[Boolean] {
+          def encode(value: Boolean): Attempt[BitVector] =
+            Attempt.successful(if (value) trueValue else falseValue)
+
+          val sizeBound: SizeBound = SizeBound.exact(8)
+
+          def decode(bits: BitVector): Attempt[DecodeResult[Boolean]] = {
+            if (bits.isEmpty) Attempt.successful(DecodeResult(false, BitVector.empty))
```

Comment:
This looks like standard boolean coded in scodec. If it is not can you point out differences why we cannot use the one from library ?

Reply:
No idea, I assumed you had some reason 😇. Will try to revert to standard one and see if tests pass.

Or would you prefer I spend the time and dig deeper?